scipy/stats/_binomtest.py

```diff
@@ -21,31 +21,42 @@ class BinomTestResult:
         Indicates the alternative hypothesis specified in the input
         to `binomtest`.  It will be one of ``'two-sided'``, ``'greater'``,
         or ``'less'``.
+    statistic: float
+        The estimate of the proportion of successes.
     pvalue : float
         The p-value of the hypothesis test.
-    proportion_estimate : float
-        The estimate of the proportion of successes.
 
     """
-    def __init__(self, k, n, alternative, pvalue, proportion_estimate):
+    def __init__(self, k, n, alternative, statistic, pvalue):
         self.k = k
         self.n = n
         self.alternative = alternative
-        self.proportion_estimate = proportion_estimate
+        self.statistic = statistic
```

Comment:
```suggestion
        self.statistic = statistic
        self.proportion_estimate = statistic
```
Why not this instead of the stuff below?
I was speaking loosely when I wrote "alias". I don't think users will be modifying them, but even if one is changed, the other does not need to change IMO. Do you think that's important for a result object, especially considering that `proportion_estimate` won't even be documented anymore?

Reply:
I guess it doesn't matter too much. Having `proportion_estimate` as a property does avoid having it appear in `var(res)`, otherwise I don't think there's much difference. I can change it if you prefer.

Reply:
@tupui thoughts on making it a property or normal attribute?

In any case, I'd keep it consistent between the three functions. I don't know that `binomtest` deserves special treatment.

Reply:
Thought I should check in again @tupui - what do you think about just making these "aliases" normal attributes rather than (more complex) properties?

Reply:
Since the result class is somehow supposed to be immutable (tuple), I think it's fine to just use an attribute for these aliases. But, ideally I think that we want to deprecate aliases, hence making a property would allow to raise a warning. Though I would only add the property and not have a setter.

Reply:
OK, for the moment I've removed the property methods and just added the alias as an attribute as @mdhaber suggested. I guess this can be converted to a property later on if a deprecation warning needs to be added.